yt_dlp/extractor/youtube.py

```diff
@@ -5699,6 +5702,22 @@ class YoutubeHistoryIE(YoutubeFeedsInfoExtractor):
     }]
 
 
+class YoutubeStoriesIE(InfoExtractor):
+    IE_DESC = 'YouTube channel stories; "ytstories:<channel id>'
+    IE_NAME = 'youtube:stories'
+    _VALID_URL = r'ytstories:(?:RLTD|UC)(?P<id>[A-Za-z0-9_-]{21}[AQgw])$'
+    _TESTS = [{
+        'url': 'ytstories:UCwFCb4jeqaKWnciAYM-ZVHg',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = f'RLTD{self._match_id(url)}'
+        return self.url_result(
+            f'https://www.youtube.com/playlist?list={playlist_id}&playnext=1',
```

Comment:
since the playlist is unviewable, we can't use the /playlist page.

playnext=1 will automatically make it redirect to the watch page of the first video in the playlist.

i.e. https://www.youtube.com/watch?v=chHM75fI770&list=RLTDwFCb4jeqaKWnciAYM-ZVHg